ComposeStarter/app/build.gradle

```diff
@@ -63,6 +63,7 @@ android {
 dependencies {
     // General compose dependencies
     implementation libs.androidx.activity.compose
+    debugImplementation libs.compose.ui.tooling
```

Comment:
What does the buildVariants view in AndroidStudio show? I think we changed the default to Release for realistic performance testing.  This might have caused this bug.

Reply:
Try flipping it in Android Studio and let me know if that fixes it.

Reply:
Flipping the setting in Android Studio does fix it, or alternatively changing "debugImplementation" to "implementation". 
Is it normal for the preview to be using a phone frame?

Reply:
Nope that's a bug.  Are you using Android Studio Chipmunk or later?

https://github.com/android/wear-os-samples/blob/7775f0269bdd52e5a68651acd319f7b504121649/ComposeStarter/app/src/main/java/com/example/android/wearable/composestarter/presentation/MainActivity.kt#L89

Reply:
I'm using Bumblebee. I'll give Chipmunk a try.